configs/defs/rocky.textproto

```diff
@@ -0,0 +1,1179 @@
+      "/home partition.\n"
+      "Example:\n"
+      "<device> /home <fstype> defaults, rw, usrquota, grpquota, nodev, relatime"
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "scan_type_specific:{instance_scanning:{"
+      "check_alternatives:{"
+      "  file_checks:{"
+      "    files_to_check:{single_file:{path:\"/proc/self/mountinfo\"}}"
+      "    content_entry:{"
+      "      match_type: ALL_MATCH_ANY_ORDER"
+      "      match_criteria: {"
+      "          filter_regex: \".* /home .*\""
+      "          expected_regex: \".* /home .*grpgquota.*\""
+      "      }"
+      "    }"
+      "  }"
+      "}"
+      "check_alternatives:{"
+      "  file_checks:{"
+      "    files_to_check:{single_file:{path:\"/proc/self/mountinfo\"}}"
+      "    content_entry:{"
+      "      match_type: NONE_MATCH"
+      "      match_criteria: {"
+      "          filter_regex: \".* /home .* - .*\""
+      "          expected_regex: \".* /home .* - .*\""
+      "      }"
+      "    }"
+      "  }"
+      "}}}"
+  }
+}
+benchmark_configs: {
+  id: "boot-rescue-mode-password"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:rockylinux:rockylinux:8.5" version: "1.0.0" benchmark_document: "CIS Rocky Linux 8" }
+    title: "Ensure authentication is required when booting into rescue mode"
+    description:
+      "Rescue mode (former single user mode) is used for recovery when the system detects an "
+      "issue during boot or by manual selection from the bootloader."
+    rationale:
+      "Requiring authenticationinrescue mode (former single user mode) prevents an "
+      "unauthorized user from rebooting the system into rescue mode to gainroot privileges "
+      "without credentials."
+    remediation:
+      "The systemd drop-in files mustbe created if it is necessary to change the default settings:\n"
+      "Create the file /etc/systemd/system/rescue.service.d/00-require-auth.conf which "
+      "contains only the configuration to be overridden:\n"
+      "[Service]\n"
+      "ExecStart=-/usr/lib/systemd/systemd-sulogin-shell rescue"
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      " check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/usr/lib/systemd/system/rescue.service\"}}"
+      "      files_to_check:{files_in_dir:{dir_path:\"/etc/systemd/system/rescue.service.d\",recursive:true}}"
+      "      content_entry:{"
+      "        match_type: ALL_MATCH_ANY_ORDER"
+      "        match_criteria: {"
+      "          filter_regex: \".*/systemd-sulogin-shell.*\""
+      "          expected_regex: \"ExecStart=/usr/lib/systemd/systemd-sulogin-shell rescue\""
+      "        }"
+      "      }"
+      "    }"
+      " }"
+      "}"
+  }
+}
+benchmark_configs: {
+  id: "systemwide-crypto-policy-is-not-legacy"
```

Comment:
Let's remove "is" from the IDs here and elsewhere to keep them short: `systemwide-crypto-policy-not-legacy`